lib/fs/casefs.go

```diff
@@ -29,7 +29,7 @@ type ErrCaseConflict struct {
 }
 
 func (e *ErrCaseConflict) Error() string {
-	return fmt.Sprintf(`given name "%v" differs from name in filesystem "%v"`, e.Given, e.Real)
+	return fmt.Sprintf(`remote "%v" is spelled in a different case than local "%v"; use identical spelling on both sides to fix`, e.Given, e.Real)
```

Comment:
```suggestion
	return fmt.Sprintf(`remote "%v" uses a different upper-/lowercase variant than local "%v"; use identical casing on both sides to fix`, e.Given, e.Real)
```

(This might be my english-as-secondary-language personality, but "spelling" doesn't sound like what I'd consider a case difference... Someone with a primary language that uses case as a significant language feature might have another opinion...)

Reply:
Well, I took the text from https://github.com/syncthing/syncthing/issues/7827#issuecomment-882494860. The idea seems to be to use layman's terms instead of more technical language. "Spelling" does seem to be used in the context of case, if you look it up in the Internet (e.g. https://www.ibm.com/support/pages/node/4931943, "The HATS Drop-down widget was matching multiple values in the list if the values were the same but were spelled using a different case."), so I'm not sure about this one. I'm not a native speaker either though.

Reply:
As a native speaker, I think
`remote "%v" uses a different upper-/lowercase variant than local "%v"; use identical casing on both sides to fix`
is fine, but perhaps we could be a little more explicit:
`remote "%v" uses a different upper-/lowercase variant than local "%v"; change the casing on either side to fix`
?

Reply:
I have reworded and shortened the message slightly. What do you think about it now?

Reply:
`uses different upper or lowercase` sounds mildly incomplete, like the word `characters` should be added. 
Without wanting to be pedantic, how about:
`remote "%v" differs only by case from local "%v"; change the casing of either file name to fix`
?

Reply:
I have added "characters" 😉. I actually like it a lot, as it gives more context to the user, especially a non-English speaking one to know what to look for. I have left the rest unchanged. I don't want to add "file" in there, as the problem may be about a folder, not a file.